### PR TITLE
No root page for system vault, redirect to managed

### PR DESCRIPTION
**Pull request by PopDaph**
## Description

Fixes https://github.com/dust-tt/tasks/issues/1283

We don't want to allow users to land on the root of the system vault, since this screen has no point as the system vault contains only managed data sources and member management is not allowed:
- We redirect to the managed category.
- We block the patch route (ceinture & bretelles).

## Risk

Can be rolled back. 

## Deploy Plan

Deploy front.

### CODE REVIEW

File: front/pages/api/w/[wId]/groups/[gId]/index.ts [hunk 1]
```diff
@@ -71,6 +71,15 @@ async function handler(
           },
         });
       }
+      if (groupRes.value.kind === "system") {
```
Comment (flvndvd) [inline front/pages/api/w/[wId]/groups/[gId]/index.ts:74]: IIRC, we have `groupRes.isSystem`.

File: front/pages/api/w/[wId]/groups/[gId]/index.ts [hunk 2]
```diff
@@ -71,6 +71,15 @@ async function handler(
...
+        return apiError(req, res, {
+          status_code: 400,
```
Comment (flvndvd) [inline front/pages/api/w/[wId]/groups/[gId]/index.ts:76]: 404 no?

File: front/pages/api/w/[wId]/groups/[gId]/index.ts [hunk 3]
```diff
@@ -71,6 +71,15 @@ async function handler(
...
+          api_error: {
+            type: "invalid_request_error",
+            message: "Can administrate the system group.",
```
Comment (flvndvd) [inline front/pages/api/w/[wId]/groups/[gId]/index.ts:79]: Wrong message?

**Review by flvndvd — APPROVED** [review]

File: front/pages/api/w/[wId]/groups/[gId]/index.ts (hunk 2 above)
Comment (PopDaph) [reply front/pages/api/w/[wId]/groups/[gId]/index.ts]: Agreed 400 not great, I've put a 403 instead!

**Pull request merged**